client/data/php-versions/index.jsx

```diff
@@ -1,10 +1,11 @@
 import { __, sprintf } from '@wordpress/i18n';
 
-export const getPHPVersions = () => {
+export const getPHPVersions = ( siteId ) => {
```

Comment:
Nitpick: I'd personally set a default value here rather than checking it below:

```suggestion
export const getPHPVersions = ( siteId = 0 ) => {
```